rotkehlchen/tests/unit/test_eth2.py

```diff
@@ -844,6 +849,168 @@ def check_performance_validator(performance, vindex, check_keys, expected_data,
+                timestamp=TimestampMS(timestamp + (36 * (HOUR_IN_SECONDS * 1000))),
+                amount=withdrawal_amount,
+                withdrawal_address=user_address,
+                is_exit=False,
+            ), EvmEvent(
+                tx_hash=make_evm_tx_hash(),
+                sequence_index=1,
+                timestamp=TimestampMS(timestamp + (40 * (HOUR_IN_SECONDS * 1000))),
+                location=Location.ETHEREUM,
+                event_type=HistoryEventType.INFORMATIONAL,
+                event_subtype=HistoryEventSubType.CONSOLIDATE,
+                asset=A_ETH,
+                amount=ZERO,
+                location_label=user_address,
+                notes='Request to consolidate validator 67929 into 1073521',
+                counterparty=CPT_ETH2,
+                address=CONSOLIDATION_REQUEST_CONTRACT,
+                extra_data={'source_validator_index': 67929, 'target_validator_index': 1073521},
+            ), EvmEvent(
+                tx_hash=make_evm_tx_hash(),
+                sequence_index=1,
+                timestamp=TimestampMS(timestamp + (40 * (HOUR_IN_SECONDS * 1000))),
+                location=Location.ETHEREUM,
+                event_type=HistoryEventType.INFORMATIONAL,
+                event_subtype=HistoryEventSubType.CONSOLIDATE,
+                asset=A_ETH,
+                amount=ZERO,
+                location_label=user_address,
+                notes='Request to consolidate validator 47086 into 1073521',
+                counterparty=CPT_ETH2,
+                address=CONSOLIDATION_REQUEST_CONTRACT,
+                extra_data={'source_validator_index': 47086, 'target_validator_index': 1073521},
+            ), EvmEvent(
+                tx_hash=make_evm_tx_hash(),
+                sequence_index=1,
+                timestamp=TimestampMS(timestamp + (40 * (HOUR_IN_SECONDS * 1000))),
+                location=Location.ETHEREUM,
+                event_type=HistoryEventType.INFORMATIONAL,
+                event_subtype=HistoryEventSubType.CONSOLIDATE,
+                asset=A_ETH,
+                amount=ZERO,
+                location_label=user_address,
+                notes='Request to consolidate validator 502836 into 1073521',
+                counterparty=CPT_ETH2,
+                address=CONSOLIDATION_REQUEST_CONTRACT,
+                extra_data={'source_validator_index': 502836, 'target_validator_index': 1073521},
+            ), EthWithdrawalEvent(
+                validator_index=validator5.validator_index,  # type: ignore[arg-type]
+                timestamp=TimestampMS(timestamp + (72 * (HOUR_IN_SECONDS * 1000))),
+                amount=FVal(32.25),
+                withdrawal_address=user_address,
+                is_exit=True,
+            ),
+        ])
+
+    performance = eth2.get_performance(
+        from_ts=Timestamp(0),
+        to_ts=ts_ms_to_sec(TimestampMS(timestamp + (100 * (HOUR_IN_SECONDS * 1000)))),
```

Comment:
we have a function like this

```suggestion
        to_ts=ts_ms_to_sec(TimestampMS(timestamp + (100 * ts_sec_to_ms(HOUR_IN_SECONDS ))))),
```